components/modal/AddToFolderModal.tsx

```diff
@@ -0,0 +1,71 @@
+import { useState, useEffect } from "react";
+import { getFolderListData } from "@/lib/api";
+import { MODALS } from "@/lib/const";
+import FolderInModal from "./FolderInModal";
+import ModalLayout from "./ModalLayout";
+import { FolderList, MODAL } from "../../types/commonTypes";
+import styles from "./AddToFolderModal.module.css";
+
+interface Props {
+  url: string;
+  handleClickModal: (value: MODAL) => void;
+  linkValue?: any;
+  makeEmptyValue?: any;
```

Comment:
any 타입은 불가피한 경우가 아니면 사용하지 말아주세요!
지금같은 경우, makeEmptyValue는 함수로 사용되고 있는데 props로 전달할 때 함수가 아닌 다른 값을 전달한다면 컴파일 단계에서 에러를 확인할 수 없고 실제 런타임에서 에러가 발생하게 되겠네요.
```suggestion
  linkValue?: string;
  makeEmptyValue?: () => void;
```